src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs

```diff
@@ -564,10 +564,49 @@ public void AcceptToMemoryPool_TxTooManyAncestors_ReturnsFalse()
         }
 
         [Fact]
-        public void AcceptToMemoryPool_TxAncestorsConflictSpend_ReturnsFalse()
+        public async void AcceptToMemoryPool_TxAncestorsConflictSpend_ReturnsFalseAsync()
         {
             // TODO: Execute failure cases for CheckAncestors
             // - conflicting spend transaction
+
```

Comment:
You can now remove the `TODO:`  or are there more scenarios in this test coming?